packages/common-utils/src/types.ts

```diff
@@ -396,6 +396,7 @@ export enum SourceKind {
   Log = 'log',
```

Comment:
Can you run `yarn changeset` and commit the changeset? since we need to bump this pkg version

Reply:
Done